flake.nix

```diff
@@ -65,7 +65,7 @@
               topiary-inplace = pkgs.writeShellApplication {
                 name = "topiary-inplace";
                 text = ''
-                  ${code.topiary-cli}/bin/topiary --in-place --input-files "$@";
+                  ${code.topiary-cli}/bin/topiary fmt "$@";
                 '';
               };
             in "${topiary-inplace}/bin/topiary-inplace";
```

Comment:
If I'm not mistaken, we can now just replace the whole thing by:
```
entry = "${code.topiary-cli}/bin/topiary fmt";
```
and we don't need to introduce `topiary-inplace` anymore. This was already the case before this PR with:
```
entry = "${code.topiary-cli}/bin/topiary --in-place --input-files";
```

Reply:
I'm not a Nix-person, so I'll take your word on this :wink: Changed in 21efefbf3cf